R/drive_update.R

```diff
@@ -6,20 +6,30 @@
 #'
 #' @template file
 #' @template media
-#' @param ... Parameters to pass along to the API query. NOT IMPLEMENTED YET.
```

Comment:
The description will need to be expanded here. Commenting on this line, just because I can.